src/k8s-engine/wsl.ts

```diff
@@ -1531,6 +1571,7 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         }
         this.process?.kill('SIGTERM');
         await Promise.all(Object.values(this.mobySocketProxyProcesses).map(proc => proc.stop()));
+        await Promise.all(Object.values(this.resolverHostProcess).map(proc => proc.stop()));
```

Comment:
```suggestion
        await this.resolverHostProcess.stop();
```
`this.resolverHostProcess` is not a `Record<T, U>`, so we shouldn't call `Object.values` on it. (It works type-wise because everything is an `Object`, but it's not what you intend here.)